packages/react-native/Libraries/WebSocket/__tests__/WebSocket-test.js

```diff
@@ -28,4 +28,25 @@ describe('WebSocket', function () {
   it('should have connection lifecycle constants defined on the instance', () => {
     expect(new WebSocket('wss://echo.websocket.org').CONNECTING).toEqual(0);
   });
+
+  it('should have binaryType of undefined when BlobManager is not available', () => {
+    expect(new WebSocket('wss://echo.websocket.org').binaryType).toEqual(
+      undefined,
+    );
+  });
+
+  // it('should have binaryType of blob when BlobManager is available', () => {
```

Comment:
Why is this commented out?

Reply:
I need help on this test case. It doesn't work currently.  What do I need to mock to make `BlobManager.isAvailable === true`?

Reply:
```diff
diff --git a/packages/react-native/Libraries/WebSocket/__tests__/WebSocket-test.js b/packages/react-native/Libraries/WebSocket/__tests__/WebSocket-test.js
index 279682c6f3d..20edae76b7d 100644
--- a/packages/react-native/Libraries/WebSocket/__tests__/WebSocket-test.js
+++ b/packages/react-native/Libraries/WebSocket/__tests__/WebSocket-test.js
@@ -18,7 +18,11 @@ jest.setMock('../../BatchedBridge/NativeModules', {
   PlatformConstants: {},
 });

-const WebSocket = require('../WebSocket');
+let WebSocket;
+
+jest.isolateModules(() => {
+  WebSocket = require('../WebSocket');
+});

 describe('WebSocket', function () {
   it('should have connection lifecycle constants defined on the class', () => {
@@ -35,18 +39,17 @@ describe('WebSocket', function () {
     );
   });

-  // it('should have binaryType of blob when BlobManager is available', () => {
-  //   jest.setMock('../../BatchedBridge/NativeModules', {
-  //     WebSocketModule: {
-  //       connect: () => {},
-  //     },
-  //     PlatformConstants: {},
-  //     BlobModule: require('../../Blob/__mocks__/BlobModule'),
-  //   });
-  //   const BlobManager = require('../../Blob/BlobManager')
-  //   expect(BlobManager.isAvailable).toEqual(true);
-  //   expect(new WebSocket('wss://echo.websocket.org').binaryType).toEqual(
-  //     'blob',
-  //   );
-  // });
+  it('should have binaryType of blob when BlobManager is available', () => {
+    jest.doMock('../../Blob/BlobManager', () => ({
+      isAvailable: true,
+      addWebSocketHandler: () => {},
+    }));
+    let WebSocket;
+    jest.isolateModules(() => {
+      WebSocket = require('../WebSocket');
+    });
+    expect(new WebSocket('wss://echo.websocket.org').binaryType).toEqual(
+      'blob',
+    );
+  });
 });
```